server/actions/Contentful.ts

```diff
@@ -45,13 +48,93 @@ export async function uploadImage(image: File){
 
 }
 
-
+/**
+* @param ID ID of the Contentful Asset to be deleted
+* @returns True if deletion is successful, false if deletion failed.
+*/
 export async function deleteAssetByID(ID: string){
-    const space = await client.getSpace(process.env.CONTENTFUL_SPACE);
-    const environment = await  space.getEnvironment('master');
-    const asset = await environment.getAsset(ID); 
-    //Before an asset can be deleted, it has to be unpublished.
-    await asset.unpublish();
-    await asset.delete();
-    console.log("Asset deleted.");
+    try{
+        const space = await client.getSpace(process.env.CONTENTFUL_SPACE as string);
+        const environment = await  space.getEnvironment('master');
+        const asset = await environment.getAsset(ID); 
+        //Before an asset can be deleted, it has to be unpublished.
+        await asset.unpublish();
+        await asset.delete();
+        console.log("Asset deleted.");
+        return true;
+    } catch (error){
+        if(error) console.error(error);
+        return false;
+    }
 }
+
+/**
+* @param journalEntry Journal Entry information to be uploaded to Contentful
+* @returns True if creation is successful, false if creation is unsuccessful.
+*/
+export async function createJournalEntry(journalEntry: JournalEntry){
+    try{
+        const space = await client.getSpace(process.env.CONTENTFUL_SPACE as string);
+        const environment = await space.getEnvironment('master');
+        const entry = await environment.createEntry('blogPost', {
```

Comment:
We may want to store these strings in a utils file and import it so that we only have to change it once if needed